components/authz-service/server/v2/rules_property_test.go

```diff
@@ -33,8 +36,10 @@ type projectAndRuleReq struct {
 var createRuleReqGen, createProjectReqGen, createProjectAndRulesGen, createProjectsAndRulesGen = getGenerators()
 
 func TestCreateRuleProperties(t *testing.T) {
-	ctx := context.Background()
-	cl, testDB, store, seed := testhelpers.SetupProjectsAndRulesWithDB(t)
+	ctx := auth_context.NewOutgoingContext(auth_context.NewContext(context.Background(),
+		[]string{SuperuserSubject}, []string{}, "*", "*", "v2.1"))
+
+	projectClient, policyClient, testDB, store, seed := testhelpers.SetupProjectsAndRulesWithDB(t)
```

Comment:
Renamed `cl` to `projectClient` because I needed to add `policyClient` as another return value. `policyClient` is needed to create the default system roles which are now needed when creating projects, because projects create policies, and those policies use roles. 😁